src/main/java/com/yelp/nrtsearch/server/luceneserver/PeriodicCommit.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Yelp Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yelp.nrtsearch.server.luceneserver;
+
+import com.yelp.nrtsearch.server.config.LuceneServerConfiguration;
+import java.io.IOException;
+import java.util.Timer;
+import java.util.TimerTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PeriodicCommit {
+
+  Logger logger = LoggerFactory.getLogger(IndexState.class);
```

Comment:
This should be `private static final` and reference the current class. Feel free to fix the one in `IndexState` too.